src/openqaoa-core/problems/minimumvertexcover.py

```diff
@@ -171,7 +156,7 @@ def terms_and_weights(self):
     def get_qubo_problem(self):
```

Comment:
Shall we change all of this to, simply, `qubo()` ?

Reply:
I'm ok with either naming.
Maybe I have a slight preference for renaming it to `qubo` and also changing this method into a property so that `ProblemClass.qubo` is a valid way to get the QUBO object.

Reply:
Sounds good to me!

Reply:
Resolved in commit [1b39d39](https://github.com/entropicalabs/openqaoa/pull/170/commits/1b39d394854bb27a07dce468dc7c81e17fc560d5)